docs/04-operation-guides/troubleshooting/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md

```diff
@@ -80,16 +80,16 @@ kyma deploy --values-file {VALUES_FILE_PATH}
       cpu: 40m
       memory: 200Mi
   ```
+  
+  > **TIP:** You should be fine with increasing the limit to 6Gi. However, if your resources are scarce, try increasing the value gradually in steps of 1Gi.
 
-> **TIP:** You should be fine with increasing the limit to 6Gi. But if your resources are scarce, try increasing the value gradually in steps of 1Gi.
-
-3. If the problem persists, drop labels for the Istio metrics, edit `prometheus-istio server`:
+3. If the problem persists, drop additional labels for the Istio metrics by editing `prometheus-istio server`:
 
   ```bash
   kubectl edit configmap -n kyma-system monitoring-prometheus-istio-server
   ```
 
-4. Edit the values:
+4. Set the following values:
```

Comment:
I'm not sure I'd change this, since `set` suggests that they haven't been set before, while `edit` clearly indicates that they are already specified in the file but must be changed.
Unless that's the intended meaning and/or this has been agreed with the devs.

Reply:
I see. Another alternative would be `modify`.